### PR TITLE
keep comments when promoting signal

### PR DESCRIPTION
**Pull request by benedikt-richter**
### Summary of changes

keep comments when promoting signal

### Context and reason for change

In the past we removed the comment of a signal when it was promoted to an attribution. This behaviour is modified to keep the comment in this case.

### CODE REVIEW

File: src/Frontend/Components/PackageCard/PackageCard.tsx [hunk 1]
```diff
@@ -105,7 +104,7 @@ export function PackageCard(props: PackageCardProps): ReactElement | null {
     <IconButton
       tooltipTitle="add"
       placement="left"
-      onClick={props.onIconClick ? props.onIconClick : doNothing}
+      onClick={props.onIconClick}
```
Comment (benedikt-richter) [inline src/Frontend/Components/PackageCard/PackageCard.tsx:107]: unrelated change

**Comment by benedikt-richter** [comment]
Waits for decision regarding versioning of the input schema.

**Review by antonbauhofer — APPROVED** [review]

**Review by leslielazzarino — APPROVED** [review]
